README.md

```diff
@@ -20,8 +20,8 @@ changed setting in the lua file. There is a playlistmanager.conf file in this re
 #### Url title resolving
 If you want playlistmanager to fetch and display titles
 of all playlist urls(mpv defaults to current file only) you will
-need to use `prefer_titles = "url" or "all"`(defaults is "url") and also use
-`titleresolver.lua` script found in this repository. The script
+need to use `prefer_titles = "url" or "all"`(defaults is "url") and also
+set the option `resolve_titles = yes`. The script
```

Comment:
Per the other change below, `resolve_titles` is only checked if `prefer_titles == "none"`, which seems weird to me. Is this setting really needed and why?

Reply:
Because prefer_titles controls if the script replaces the entry of the current file with it's title. Whereas resolve_titles calls youtube-dl on every item in the playlist to find all the url titles.

These are distinct behavioral differences, which is why I put it behind an additional config entry.

Reply:
Yeah it's a bit confusing but prefer_titles is not connected to resolving titles. I guess `prefer_titles = none` and `resolve_titles = yes` config is a bit weird since it resolves titles but never shows them. There could be some misconfiguration error in that case.